src/Api/Fields/ReportApiFields.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+namespace Silktide\ProspectClient\Api\Fields;
+
+use DateTime;
+use Psr\Http\Message\UriInterface;
+
+class ReportApiFields extends AbstractApiFields
+{
+    /** Pass values to set as one of your custom report fields. */
+    public function customField(string $key, string $value): void
```

Comment:
This is probably just me / stylistic but I feel like the word "set" is missing in all these function names?